rialto_airflow/dags/harvest.py

```diff
@@ -147,7 +147,12 @@ def fill_in_pubmed(snapshot):
         """
         Fill in Pubmed data for DOIs from other publication sources.
         """
-        pubmed.fill_in(snapshot)
```

Comment:
I think commenting this out here will make it look like the task is still running? How about instead we comment out the call to `fill_in_pubmed(snapshot)` in `fill_in`. Then it will just not be present in the workflow?

Reply:
changed